guides/common/modules/proc_salt-salt-example.adoc

```diff
@@ -0,0 +1,51 @@
+[id="salt_guide_salt_example_{context}"]
+= Salt Example
+
+This example uses a Salt State to manage the `/etc/motd` file on one or more Salt Minions.
+It demonstrates the use of {Project} as an external node classifier and the use of Salt Grains.
+
+.Procedure
+. Create a global parameter called `vendor_name` with the string `{Project}` as its value.
+. Add a new Salt State called `motd` to your Salt Master.
+.. Create the `/srv/salt/motd/` directory.
+.. Add two files to it:
+... The `/srv/salt/motd/init.sls` file is a Salt State file:
```

Comment:
We never go this deep. 
You know, I see no value in indenting substeps if the user has to do them sequentially. 
I would appreciate if there isn't three levels of indentation. I'd rather no sub steps, since it looks to me that the user has to work through these all, so the hierarchy follows the UI rather the user, and we are writing user-oriented docs. 
Up to you though.

Reply:
Good call, fixed.